products.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link href="https://fonts.googleapis.com/css?family=Arsenal&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Fredericka+the+Great&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="src/main.css">
+    <link rel="stylesheet" href="src/products.css">
+    <title>Quality Quidditch Cottage Products</title>
+</head>
+<body>
+    <main>
+
+        <section class="header-image">
+
+            <header>
+                <h1 id="main-product-cover">
+                    The Quality Quidditch Cottage
+                </h1>
+            </header>
+
+            <p class=product-page-logo>
+                <img class="title-image" src="assets/nimbus-title.png">
+            </p>
+
+            <nav class="nav-menu">
+                <a href="index.html">Home |</a>
+                <a href="products.html">Products |</a>
+                <a href="shopping-cart.html">Shopping Cart |</a>
+                <a href="product-entry.html">Create New Product |</a>
+                <a href="sales-report.html">Sales Report</a>
+            </nav>
+            
+            <h2 class="header-two">Quidditch Gear</h2>
+
+        </section>
+
+        <section id="center-section">
+
+                <div id="menu-outer">
+                    <div class="table">
+                        <ul id="gear">
+                        </ul>
+                    </div>
+                </div>
+
+        </section>
+
+    </main>
+
+    <script type="module" src="/src/data/quidditch.js"></script>
+    <script type="module" src="/src/products.js"></script>
+    <script type="module" src="/src/product-entry.js"></script>
```

Comment:
this product-entry script tag is causing a console error when you click on the "products" link at the top. If you take it out, the error will go away. You don't need it in this file.